docs/book/kubernetes-admission-control.md

```diff
@@ -0,0 +1,323 @@
+# Kubernetes Admission Control
+
+Kubernetes [Admission Controllers](https://kubernetes.io/docs/admin/admission-controllers/) perform *semantic validation* of resources during create, update, and delete operations. In Kubernetes 1.7, you can use OPA to enforce custom policies without recompiling or reconfiguring the Kubernetes API server by leveraging [External Admission Webhooks](https://kubernetes.io/docs/admin/extensible-admission-controllers/#external-admission-webhooks).
+
+## Goals
+
+This tutorial shows how to use OPA to enforce custom policies on resources in
+Kubernetes. For the purpose of this tutorial, you will define a policy that
+prevents users from running `kubectl exec` on "privileged" containers in the
+"production" namespace.
+
+## Prerequisites
+
+This tutorial requires a Kubernetes 1.7 (or later) cluster. To test Kubernetes locally, we recommend using [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/). Keep in mind that External Admission Webhook support in Kubernetes is currently in **alpha**.
+
+External Admission Controllers must be secured with TLS. See [Generating TLS Certificates](https://github.com/open-policy-agent/kube-mgmt#generating-tls-certificates)
+for steps to generate a CA and client/server credentials for test purposes.
+
+```bash
+ADMISSION_CONTROLLERS=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds,GenericAdmissionWebhook
+
+minikube start --kubernetes-version v1.7.0 \
+    --extra-config=apiserver.Admission.PluginNames=$ADMISSION_CONTROLLERS
+    --extra-config=apiserver.ProxyClientCertFile=</path/to/client.crt>
+    --extra-config=apiserver.ProxyClientKeyFile=</path/to/client.key>
+```
+
+> `minikube` automatically mounts your home directory into the VM. If you store
+> the TLS credentials under your home directory you can specify them as
+> `$PWD/client.crt` and `$PWD/client.key` above.
+
+## Steps
+
+### 1. Deploy OPA on top of Kubernetes
```

Comment:
Any reason that "Start kubernetes with an external admission controller" isn't the first step? Seems like the minikube config example would fit here better than above